pkgs/development/python-modules/git-url-parse/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pbr
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "git-url-parse";
+  version = "1.2.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "coala";
+    repo = "git-url-parse";
+    rev = version;
+    hash = "sha256-+0V/C3wE02ppdDGn7iqdvmgsUwTR7THUakUilvkzoYg=";
+  };
+
+  # Manually set version because prb wants to get it from the git
+  # upstream repository (and we are installing from tarball instead)
+  PBR_VERSION = version;
+
+  propagatedBuildInputs = [
+    pbr
+  ];
+
+  pythonImportsCheck = [ "giturlparse" ];
+
+  nativeCheckInputs = [
+    pytest
```

```suggestion
    pytestCheckHook
```
Would you please use `pytestCheckHook`?

---

eh sorry, I should have prepared these a bit more sequentially properly incorporating the learnings along the way. :handshake:

---

Don't worry, I look forward to seeing your great progress.🥳